design.md

```diff
@@ -0,0 +1,157 @@
+---
```

Comment:
Please include what version of TLS, cipher suites etc, you will support and why

Reply:
Addressed: https://github.com/rexposadas/teleport/pull/1/commits/e8d21514becd922e5aaec7ca8d840d5cd3e4180c